src/Result/AuthenticationResult.php

```diff
@@ -83,12 +83,12 @@ public function getRefreshToken(): ?string
         return $this->refreshToken;
     }
 
-    public function getExpiresAt(): ?DateTimeImmutable
+    public function getExpiresAt(): DateTimeImmutable|false
     {
         try {
-            return DateTimeImmutable::createFromFormat('U', $this->expiresIn);
+            return DateTimeImmutable::createFromFormat('U', strval($this->expiresIn));
```

Comment:
If you run `composer cs-check`, it will complain about using this function without importing it.
Please run that command and fix any other errors reported.